examples/restify/src/integration-test/groovy/restify/BulkRestApiSpec.groovy

```diff
@@ -1,22 +1,21 @@
 package restify
 
+import gorm.tools.job.JobState
 import gorm.tools.rest.JsonParserTrait
 import gorm.tools.rest.client.OkHttpRestTrait
 import grails.gorm.transactions.Rollback
 import grails.testing.mixin.integration.Integration
 import okhttp3.Response
 import org.springframework.http.HttpStatus
-import spock.lang.Ignore
 import spock.lang.Specification
+import yakworks.rally.job.Job
 import yakworks.rally.orgs.model.Org
 
 @Rollback
 @Integration
-
 class BulkRestApiSpec extends Specification implements OkHttpRestTrait, JsonParserTrait {
     String path = "/api/rally/org/bulk?jobSource=Oracle"
```

Comment:
jobSource should not be passed in, only jobSourceId.
we need to test that job.source get populated with the url (controller/action? is what got mocked in) that called it

Reply:
@Josh forgot to tell you today that I changed source with sourceid:
sourceId -- url controller/action
source - param jobSource, like Oracle -- /api/ar/customer/bulk?jobSource=Oracle

Reply:
Test updated